### ISSUE TITLE
Detect Clones configuration in sourcery.yaml

### ISSUE THREAD

**Issue by closedLoop** [sourcery-ai/sourcery#134]
### Issue description or question

I would like to configure the VSCode Detect Clones functionality to only return clones that match the following properties:
* `min-lines`
* `min-duplicates`
* `identical-clones-only`

Currently the program is discovering dozens of duplicates that only 3 or 4 lines long across 2 or 3 files. VS Code doesn't allow the Problems tab to be sorted.  

I am not sure if this is a documentation problem: https://github.com/sourcery-ai/sourcery/wiki/Sourcery-configuration
or a feature request as I have seen "Detect Clones" listed as beta.

### Sourcery Version

v0.9.0

### Code editor or IDE name and version

VSCode - 1.54.3

### OS name and version

Linux - Ubuntu 20.04.2 LTS

Current `.sourcery.yaml` 
```
ignore:
  - venv/*
  - .vscode/*

refactor:
  skip: []

metrics:
  quality_threshold: 25.0

github:
  labels: []
  ignore_labels: [sourcery-ignore]
  request_review: author
  sourcery_branch: sourcery/{base_branch}
```

**Comment by closedLoop** [comment]
Additionally I would like the ability to ignore certain files or folders (such as test folders) that often have much more boilerplate / duplicate code.

**Comment by Hellebore** [comment]
These aren't currently supported in configuration, but are all good ideas!

**Comment by Hellebore** [comment]
New config now available in 0.9.1
